features/target_specific_scenarios_by_tag.feature

```diff
@@ -45,6 +45,13 @@ Feature: Target specific scenarios
     Then it fails
     And it runs the scenario "first scenario"
 
+  Scenario: merge multiple tag expressions
+    When I run cucumber.js with `--tags @b --tags @d`
```

Comment:
I think this test would pass without your source code change (which is why its important to write your tests first) because `second scenario - Z` is the only scenario with `@d`. Thus without the repeatable option, I expect this to pass. Maybe switch to `@a` and `@d`?